9.yaml

```
@@ -0,0 +1,52 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: q9
+  labels:
+    question.kontest.amsy.dev: "9"
+data:
+  question: |-
+    永続化領域として確保されるVolumeが、PersistentVolumeです。
+    NFSを使用してcndt-pvという名前のPersistentVolumeを作成し、NFSサーバのIPを192.168.9.8、容量を8Gi、アクセスモードをRXW、Reclaim PolicyをRetain、パスを/var/lib/cndtとして以下マニフェストファイルに変更してください。
```

```suggestion
    NFSを使用してcndt-pvという名前のPersistentVolumeを作成し、NFSサーバのIPが192.168.9.8、容量が8Gi、アクセスモードがRXW、Reclaim PolicyがRetain、パスが/var/lib/cndtとなるように以下マニフェストファイルを変更してください。
```

`マニフェストファイルに変更してください。` だと日本語がおかしいのと、マニフェストファイルに変更を加えてくださいだと他と揃わない感じだったので。